src/com/qamp/HarisJasarevic/homeworks/homework3/task2/Vehicle.java

```diff
@@ -0,0 +1,29 @@
+package com.qamp.HarisJasarevic.homeworks.homework3.task2;
+
+public class Vehicle {
+
+    protected double speed;
+    protected final String model;
```

Comment:
It would be better if these were private, with protected getter methods if you only need to red these values.